### PR TITLE
Ensure EKS is compatible with pulumi-aws 5.0.0 onwards

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/go.sum [hunk 1]
```diff
@@ -161,13 +164,20 @@ github.com/prometheus/common v0.4.0/go.mod h1:TNfzLD0ON7rHzMJeJkieUDPYmFC7Snx/y8
 github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
 github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsTZCD3I8kEA=
 github.com/prometheus/tsdb v0.7.1/go.mod h1:qhTCs0VvXwvX/y3TZrWD7rabWM+ijKTux40TwIPHuXU=
-github.com/pulumi/pulumi-aws/sdk/v4 v4.0.0 h1:hotld6lXp7a1LiPsAgcp3BrGgvj3O/IB5QzdPukyC34=
-github.com/pulumi/pulumi-aws/sdk/v4 v4.0.0/go.mod h1:TCdXM+R0dW0CFco+lb3Q7qRYWmYxDbJ7EPWYzRVHWMw=
-github.com/pulumi/pulumi-kubernetes/sdk/v3 v3.0.0 h1:Z97mMkrA3pHLpysmhiQGN4O7cxJ4uPIUCiAcm8nGOe8=
-github.com/pulumi/pulumi-kubernetes/sdk/v3 v3.0.0/go.mod h1:Wgy4/LfbBAYwdFDQ1Cin5SUqLHuQ9qOGWV0ECabr+Bw=
-github.com/pulumi/pulumi/sdk/v3 v3.0.0 h1:zkragE05t1Rco/ymfqMU4UXdEmKMmOH0SXhUKQxujxQ=
-github.com/pulumi/pulumi/sdk/v3 v3.0.0/go.mod h1:GBHyQ7awNQSRmiKp/p8kIKrGrMOZeA/k2czoM/GOqds=
+github.com/pulumi/pulumi-aws/sdk/v5 v5.1.0 h1:58ixFNJToq8+uXykudWSw/UgYf5bwczQGcC5/rZdtVs=
```
Comment (viveklak) [inline sdk/go.sum:167]: Tests seem to be failing with go.sum issues. Are we making sure we are go 1.16 compatible still here?

**Comment by viveklak** [comment]
How do we plan on messaging the breaking change since this is still pre-1.0?

**Comment by davidroth** [comment]
Is it planned to add support for [ClusterKubernetesNetworkConfig-> ipFamily](https://www.pulumi.com/registry/packages/aws/api-docs/eks/cluster/#clusterkubernetesnetworkconfig) in this MR?

**Comment by viveklak** [comment]
@stack72 what is the status of this PR? Do we want to push this in? Sounds like this would be a prerequisite for #684 (enabling IPv6 support).

**Comment by stack72** [comment]
@viveklak we had a failing Python test that didn't make any sense so we had to shelf it for awsx work

Let's talk about this

P.

**Comment by viveklak** [comment]
> @viveklak we had a failing Python test that didn't make any sense so we had to shelf it for awsx work
> 
> Let's talk about this
> 
> P.

Seems to be consistent outside of this PR: https://github.com/pulumi/pulumi-eks/issues/697

**Review by viveklak — APPROVED** [review]

**Pull request merged**